.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,31 @@
+## Overview
+
+< A brief description of the pull request >
+
+---
+
+| Q             | A
+| ------------- | ---
+| Bug fix?      | yes/no
```

Comment:
I think it's a good idea to remind people to use labels, but I would like to minimize the amount of text they have to delete from the PR so I put this in with the overview help text.